index.html

```diff
@@ -182,12 +204,18 @@ <h2>Congrats, you successfully deployed a container image to Cloud Run</h2>
 		{{end}}
 
 		<div class="details">
-			<p>This created the revision <span class="code">{{.Revision}}</span> of the Cloud Run service <span
+		<p>This created the revision <span class="code">{{.Revision}}</span> of the Cloud Run service <span
 					class="code">{{.Service}}</span>{{if .ProjectFound}} in the GCP project <span
 					class="code">{{.Project}}</span>{{end}}.</p>
-
-			<p class="callout">
-				<label><img src="/assets/lightbulb_icon.svg" style="margin:0px 8px 0px 0px; vertical-align:text-top;">
+					
+		<!-- Hard coded for testing -->
+		<!--	<p>This created the revision <span class="code">hello-celebration-00001-yoc</span> of the Cloud Run service <span
+						class="code">hello-celebration</span> in the GCP project <span
+						class="code">serverless-ux-playground</span></p> -->
+	
+
+		<p class="callout">
```

Comment:
Also revert. The indentation is incorrect and you are not adding anything here as far as I can see.